ui/components/app/asset-list/asset-list.js

```diff
@@ -94,6 +94,7 @@ const AssetList = ({ onClickAsset }) => {
           tokenSymbol={primaryCurrencyProperties.suffix}
           secondary={showFiat ? secondaryCurrencyDisplay : undefined}
           tokenImage={balanceIsLoading ? null : primaryTokenImage}
+          className="native-token"
         />
```

Comment:
Instead of passing this `native-token` and needing to account for that, can there be a `boxProps` prop or something where we add the spacing? We shouldn't be passing arbitrary classnames for JS purposes.

Reply:
Hmm! Unfortunately no. `MultichainTokenListItem` is the same component that we are using to show native token and detected tokens. But this marginTop is required only for the first token cell which in this case is the native token of the network hence we need to add this margin conditionally or add it with CSS. Another approach that I can think of is to add a `margin-bottom` for the first tab using BoxProps since I had previously updated to use Box as a parent component for tab. Updated here https://github.com/MetaMask/metamask-extension/pull/18988/commits/f66dc0af9eaefbfa950789fafb1dd77cbfe976ee .